src/utils/HTTPCode/index.ts

```diff
@@ -0,0 +1,6 @@
+/** Common HTTP Codes */
+enum HTTPCode {
+  INTERNAL_SERVER_ERROR = 500,
```

Comment:
do we need the other codes?

Reply:
Probably but we can just add them as they become needed